Triangle: explicitly exclude degenerate triangles (#2579)
exercism/problem-specifications: exercises/triangle/description.md

For a shape to be a triangle at all, all sides have to be of length > 0, and the sum of the lengths of any two sides must be greater than or equal to the length of the third side.

~~~~exercism/note

> We opted to not include degenerate triangles (triangles that violate these rules) in the tests to keep things simpler.

Reviewer: ```suggestion
We opted to not include tests for degenerate triangles (triangles that violate these rules) to keep things simpler.
```
